### DISCUSSION TITLE
4.0.2

### DISCUSSION THREAD

**Discussion by timmo001** [timmo001/system-bridge#2887] [General]
Welcome to a new major version of System Bridge! Version 4 contains a number of internal changes, speed and reliability improvements.

There have not been any user facing functionality changes, however the application has been refactored to a stage where this version is not compatible with older versions, so a 4.x.x release was required. 

## 🏠 Home Assistant Integration

Since this is a major version release, the changes are not compatible with the current version of the Home Assistant integration. Please wait for an update containing [this](https://github.com/home-assistant/core/pull/107957) change before upgrading to System Bridge version 4+.

## What's Changed

* Bump systembridgefrontend from 3.10.1 to 3.11.0 by @dependabot in https://github.com/timmo001/system-bridge/pull/2866
* Bump systembridgefrontend from 3.11.0 to 3.11.1 by @dependabot in https://github.com/timmo001/system-bridge/pull/2868
* 4.0.0 by @timmo001 in https://github.com/timmo001/system-bridge/pull/2873
* Update to systembridgebackend 4.0.0 by @timmo001 in https://github.com/timmo001/system-bridge/pull/2884
* Bump systembridgebackend from 4.0.1 to 4.0.2 by @dependabot in https://github.com/timmo001/system-bridge/pull/2886

**Full Changelog**: https://github.com/timmo001/system-bridge/compare/3.11.2...4.0.2

<hr /><em>This discussion was created from the release <a href='https://github.com/timmo001/system-bridge/releases/tag/4.0.2'>4.0.2</a>.</em>

**Comment by Oxcar7** [comment]
Hey There!  Thanks for the great work on this application.  Do you have a sense of when the Home Assistant integration will be updated to be compatible with version 4?
